### ISSUE TITLE
[DECISION] Gatherplots 20-Nov-2023

### ISSUE THREAD

**Issue by codementum** [journalovi/2023-park-gatherplots#17]
## Review Links
Reviewed Version: 0c1a8b893097f65347f0e200c6b8850b91b6b8cb
R1 [#7](https://github.com/journalovi/2023-park-gatherplots/issues/7)
R2 [#15](https://github.com/journalovi/2023-park-gatherplots/issues/15)
R3 [#16](https://github.com/journalovi/2023-park-gatherplots/issues/16)
Accessibility [#6](https://github.com/journalovi/2023-park-gatherplots/issues/6)

Reviewers were generally positive about the GatherPlots direction and evaluation approach, but also offered several concrete suggestions for revisions.
Revisions that should be emphasized for the next round of review include:

## Novelty
All reviews noted issues with the novelty claims of the work ([#16](https://github.com/journalovi/2023-park-gatherplots/issues/16), [#15](https://github.com/journalovi/2023-park-gatherplots/issues/15), [#7](https://github.com/journalovi/2023-park-gatherplots/issues/7)).
The main reason noted is a more recent work called Atom, which seems to have built on Gatherplots.
While other novelty considerations (listed below) are important to address, the relation to Atom seems most pressing.

One suggestion from [#7](https://github.com/journalovi/2023-park-gatherplots/issues/7) as an approach to addressing novelty issues is to revise the submission to talk more about how Gatherplots feeds into Atom explicitly. Atom seems to have an overlapping set of authors, and because JoVI emphasizes rigor over novelty, this approach would be appropriate. For example, are there any core ideas that Gatherplots explored that were generalized in Atom, or approaches that did not make it? Such a retrospective could prove particularly useful for readers while also addressing review issues with novelty claims in the paper as written.

There are several other considerations in individual reviews that should be weighed in revising the novelty of the work.
For example, [#7](https://github.com/journalovi/2023-park-gatherplots/issues/7) mentions the scatterplot task typology from Sarikaya and Gleicher, while [#16](https://github.com/journalovi/2023-park-gatherplots/issues/16) sees possible comparisons to Rzeszotarski and Kittur.
I encourage the authors consider suggestions in the individual reviews in the revision, as well.

## Algorithm Specifics
Reviews raise specific concerns about the algorithmic and mathematical details of the work.
[#15](https://github.com/journalovi/2023-park-gatherplots/issues/15) gives suggestions for clarifying the bin sizing approach.
[#7](https://github.com/journalovi/2023-park-gatherplots/issues/7) provides several suggestions on the mathematics and geometry of the approach.
[#16](https://github.com/journalovi/2023-park-gatherplots/issues/16) suggests reducing the mathematical notation to avoid confusion.
Authors might consider several routes to addressing these concerns, either through strengthening/defending the notations or by tightening/transforming them, e.g. changing to natural language.

## Evaluation Improvements
Reviews also mention possible improvements to the evaluation.
See [#15](https://github.com/journalovi/2023-park-gatherplots/issues/15) for several suggestions, including a need for a few additional experiment details and open materials (also a main issue, below).
[#16](https://github.com/journalovi/2023-park-gatherplots/issues/16) also provides suggestions such as adding additional competitors, or at least discussing how the technique relates to competitors.
Two approaches may suffice here. Either existing claims and writing could be tightened and made more precise, or new experiments could be added to satisfy claims as they currently are given in the paper. 

## JoVI-required Materials
Reviews noted that evaluation materials such as the experiment code, data, and analysis scripts were not available.
As including such materials are part of JoVI guidelines, these should be included in a revision.
In particular, these would help with reviewers such as [#15](https://github.com/journalovi/2023-park-gatherplots/issues/15) who were attempting to replicate the experiment analysis.

## GatherLens Clarification
The GatherLens component appears to be an addition to the core of the work.
Reviews were generally not positive about its relation to the core GatherPlot narrative.
One possible way of including it in the submission, however, would be to clearly specify that GatherLens is meant to be a demonstration of how GatherPlots can be composed and used in practice, rather than claiming the GatherLens as a core contribution in itself.

## Accessibility Review + Changes
The accessibility review in issue [#6](https://github.com/journalovi/2023-park-gatherplots/issues/6) highlights several needed and several optional changes for the submission.
In general, aspects that would impede accessibility on a Desktop should be addressed.
These include having desktop legible font sizes, especially for figures.
The submission figures could be improved in this regard.
All images should also have appropriate alt text.

Other suggestions, such as label density, are optional as they are not necessarily needed to describe the purpose of the graph.
In other words, some visualizations may be illustrating the technique, rather than meaning to show the underlying data and axis values.

Finally, some suggestions may be limitations of the Quarto platform itself. These do not need to be addressed at this time as they may be difficult or impossible to fix.

Decision: Major Revision

**Comment by nickelm** [comment]
> Reviewers were generally positive about the GatherPlots direction and evaluation approach, but also offered several concrete suggestions for revisions.

The suggestions for revisions were very helpful.

> Revisions that should be emphasized for the next round of review include:

> Novelty
> All reviews noted issues with the novelty claims of the work (#16, #15, #7).
> The main reason noted is a more recent work called Atom, which seems to have built on Gatherplots.
While other novelty considerations (listed below) are important to address, the relation to Atom seems most pressing.

**Revision:** We have carefully addressed this point in the paper in the subsection "Data-aware Methods".

> One suggestion from #7 as an approach to addressing novelty issues is to revise the submission to talk more about how Gatherplots feeds into Atom explicitly. Atom seems to have an overlapping set of authors, and because JoVI emphasizes rigor over novelty, this approach would be appropriate. For example, are there any core ideas that Gatherplots explored that were generalized in Atom, or approaches that did not make it? Such a retrospective could prove particularly useful for readers while also addressing review issues with novelty claims in the paper as written.

We hope that this new revision will address novelty concerns better.

> There are several other considerations in individual reviews that should be weighed in revising the novelty of the work.
> For example, #7 mentions the scatterplot task typology from Sarikaya and Gleicher, while #16 sees possible comparisons to Rzeszotarski and Kittur.
> I encourage the authors consider suggestions in the individual reviews in the revision, as well.

Agreed. We have addressed each of these concerns from individual reviews in turn.

**Revision:** We have improved the background section and the comparison to prior art. In particular, we have added more recent missing work from our initial writing of this article.

> Algorithm Specifics
> Reviews raise specific concerns about the algorithmic and mathematical details of the work.
> #15 gives suggestions for clarifying the bin sizing approach.

**Revision:** We have added details on bin sizing.

> #7 provides several suggestions on the mathematics and geometry of the approach.

**Revision:** We have discussed and adopted these suggestions.

> #16 suggests reducing the mathematical notation to avoid confusion.

**Revision:** We have removed the unnecessary mathematical formalism and replaced it with natural language.

> Authors might consider several routes to addressing these concerns, either through strengthening/defending the notations or by tightening/transforming them, e.g. changing to natural language.

These were all good points of feedback, and I believe that I have addressed them all. 

> Evaluation Improvements
> Reviews also mention possible improvements to the evaluation.
> See #15 for several suggestions, including a need for a few additional experiment details and open materials (also a main issue, below).

**Revision:** Fixed.

> #16 also provides suggestions such as adding additional competitors, or at least discussing how the technique relates to competitors.

We opted not to do this; see our response in issue #16. 

> Two approaches may suffice here. Either existing claims and writing could be tightened and made more precise, or new experiments could be added to satisfy claims as they currently are given in the paper.

We have opted for the latter, improving the writing and claims as well as the reporting of the results.

> JoVI-required Materials
> Reviews noted that evaluation materials such as the experiment code, data, and analysis scripts were not available.
> As including such materials are part of JoVI guidelines, these should be included in a revision.
> In particular, these would help with reviewers such as #15 who were attempting to replicate the experiment analysis.

**Revision:** Thanks for your patience; I believe that I have added all of the required materials to the OSF. (I still need to register the GitHub files, but will only do this once my final changes are pushed.)

> GatherLens Clarification
> The GatherLens component appears to be an addition to the core of the work.
> Reviews were generally not positive about its relation to the core GatherPlot narrative.
> One possible way of including it in the submission, however, would be to clearly specify that GatherLens is meant to be a demonstration of how GatherPlots can be composed and used in practice, rather than claiming the GatherLens as a core contribution in itself.

**Revision:** I ended up dropping both the section and the contribution. I am somewhat enamored by the application, but since there is no interactive version to demonstrate it may be best left outside the paper.

> Accessibility Review + Changes
> The accessibility review in issue #6 highlights several needed and several optional changes for the submission.
> In general, aspects that would impede accessibility on a Desktop should be addressed.
> These include having desktop legible font sizes, especially for figures.
> The submission figures could be improved in this regard.
> All images should also have appropriate alt text.

**Revision:** I am working with the accessibility reviewer/editor to fix this. I submitted one revision and will tackle the updates soon.

> Other suggestions, such as label density, are optional as they are not necessarily needed to describe the purpose of the graph.
> In other words, some visualizations may be illustrating the technique, rather than meaning to show the underlying data and axis values.

Sounds good, thank you.

> Finally, some suggestions may be limitations of the Quarto platform itself. These do not need to be addressed at this time as they may be difficult or impossible to fix.

Understood.

> Decision: Major Revision

Thank you for the careful feedback from both the editor and the individual reviewers. Also thank you for your patience. I hope this new version will be a step closer.
